### DISCUSSION TITLE
How to support this project

### DISCUSSION THREAD

**Discussion by oaklandgit** [replit/kaboom#177] [General]
Aside from contributing code, is there a way to support this project financially? (e.g. Patreon?) or is this already funded by replit?

Thanks

**Comment by lajbel** [comment]
You can help by buying a Hacker subscription at [replit.com](https://replit.com/pricing). [View this issue](https://github.com/replit/kaboom/issues/94)

**Reply by lajbel** [reply]
(In the same way, I think it is financed by replit)

**Reply by oaklandgit** [reply]
Indeed I will!

**Comment by slmjkdbtl** [comment]
Yep Replit is paying me now! The best way I can think of is provide feedbacks, feature requests and bug reports, and spread the words!
